R/DatabaseConnector.R

```diff
@@ -26,6 +26,21 @@ NULL
 
 .onLoad <- function(libname, pkgname) {
   rJava::.jpackage(pkgname, jars = "DatabaseConnector.jar", lib.loc = libname)
+  rJava::.jpackage(pkgname, jars = "api-common-1.8.1.jar", lib.loc = libname)
```

Comment:
This is not how the .jpackage function works. Please read the rJava documentation

Reply:
thank you point this 

> I have serious concerns about adding this many external jar files. Please make a case-by-case argument for each jar file why it absolutely has to be included. For those that must be included, they should go into the DatabaseConnectorJars package.
> 
> Also, please target the develop branch.

all this jar are necessary just to activate the stream  data loading functionality by big query: https://cloud.google.com/bigquery/streaming-data-into-bigquery
We are run a few analyses and found that it is not so easy to delete table afterword 
https://stackoverflow.com/a/43086239/1167673. So I am going to close this pull request and create another one.